src/Events/Integrations/Modules/Ticket_Emails/Email/RSVP.php

```diff
@@ -133,47 +133,75 @@ public function include_attachments( $attachments, $email_id, $email_class ) {
 	 * @return void
 	 */
 	public function include_event_links( $parent_template ) {
-		$email_class = tribe( RSVP_Email::class );
+		if ( ! $this->should_show_links( $parent_template, $email_class, $args ) ) {
+			return;
+		}
 
-		// Bail early if the email class is not enabled.
-		if ( ! $email_class->is_enabled() ) {
+		$args['event_gcal_link'] = tribe( Google_Calendar::class )->generate_single_url( $args['event']->ID );
+		$args['event_ical_link'] = tribe_get_single_ical_link( $args['event']->ID );
+
+		if ( ! empty( $args['preview'] ) ) {
+			$args['event_gcal_link'] = '#';
+			$args['event_ical_link'] = '#';
+		}
+
+		tribe( Template::class )->template( 'template-parts/body/event/links', $args, true );
+	}
+
+	/**
+	 * Maybe include event link styles.
+	 *
+	 * @since TBD
+	 *
+	 * @param \Tribe__Template $parent_template Event Tickets template object.
+	 *
+	 * @return void
+	 */
+	public function include_event_link_styles( $parent_template ): void {
+		if ( ! $this->should_show_links( $parent_template, $email_class, $args ) ) {
 			return;
 		}
 
-		$use_ticket_email = tribe_get_option( $email_class->get_option_key( 'use-ticket-email' ), false );
+		tribe( Template::class )->template( 'template-parts/header/head/tec-style', $args, true );
```

Comment:
For consistency, I'd go with `styles`.

Reply:
When I use `styles` it overrides the `styles` file from ET. That's why I named it differently. Or are you suggesting I use `tec-styles`?

Reply:
I renamed it `tec-styles`